test/formal/combine_div_shl_one_32.py

```diff
@@ -0,0 +1,27 @@
+from rule import Rule
+from ...common.opcodes import *
```

Comment:
This is wrong now, isn't it?

Reply:
ugh yes, was trying stuff and forgot to remove

Reply:
updated